Formula/ck.rb

```diff
@@ -0,0 +1,28 @@
+class Ck < Formula
+  desc "Concurrency primitives and non-blocking data structures library"
+  homepage "http://concurrencykit.org/"
+  url "http://concurrencykit.org/releases/ck-0.6.0.tar.gz"
+  sha256 "d7e27dd0a679e45632951e672f8288228f32310dfed2d5855e9573a9cf0d62df"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
```

Comment:
Does the `configure` script support `--disable-dependency-tracking`? If so, add it.